server/stream.go

```diff
@@ -1139,6 +1139,10 @@ func (mset *stream) processMirrorMsgs() {
 
 	// Grab stream quit channel.
 	mset.mu.Lock()
+	if mset.mirror == nil {
```

Comment:
and is it possible that mset.mirror.msgs be nil? because if so, then you would get a panic also because of that.

Reply:
we don't apparently.

Reply:
I double checked and we are good, I just nil out mset.mirror.